config/translations/ru/database.dump.yml

```diff
@@ -1,7 +1,7 @@
-description: 'Дамп структуры и контента MySQL БД и таблиц'
+description: 'Дамп структуры и содержимого базы данных'
```

Comment:
@enzolutions that translation was wrong, there is not mention about MySQL in the https://github.com/hechoendrupal/DrupalConsole/blob/master/config/translations/en/database.dump.yml